Domain.Sql.Tests/SqlEventSourcedRepositoryTests.cs

```diff
@@ -98,14 +110,15 @@ protected override IStoredEvent CreateStoredEvent(string streamName, string type
                     StreamName = "Order",
                     Type = "UKNOWN",
                     Body = new { ShoeSize = 10.5 }.ToJson(),
-                    SequenceNumber = 1,
+                    SequenceNumber = 2,
                     AggregateId = orderId,
```

Comment:
No particularly good reason. Was testing EventSequence code paths when fixing the test, changed it to 2 at that time.